src/safeds_runner/server/main.py

```diff
@@ -0,0 +1,202 @@
+"""Module containing the main entry point, for starting the Safe-DS runner."""
+
+import argparse
+import json
+import logging
+import typing
+from typing import Any
+
+import flask.app
+import flask_sock
+import simple_websocket
+from flask import Flask
+from flask_cors import CORS
+from flask_sock import Sock
+
+from safeds_runner.server import messages
+from safeds_runner.server.messages import create_placeholder_value, message_type_placeholder_value
+from safeds_runner.server.pipeline_manager import (
+    execute_pipeline,
+    get_placeholder,
+    set_new_websocket_target,
+    setup_pipeline_execution,
+)
+
+
+def create_flask_app(testing: bool = False) -> flask.app.App:
+    """
+    Create a flask app, that handles all requests.
+
+    :param testing Whether the app should run in a testing context
```

Comment:
Docstrings should use the Numpydoc style:

* [Specification](https://numpydoc.readthedocs.io/en/latest/format.html)
* [Project setup (4. item)](https://runner.safeds.com/en/stable/development/environment/#project-setup)
* [Project guidelines](https://library.safeds.com/en/stable/development/project_guidelines/#docstrings) (other repo, will be copied here too)